#goal definition
float32 x_cordinate
float32 y_cordinate
float32 x_center
float32 y_center

---
#result definition
string name
---
#feedback
string[] flag

